spec/app_size_metrics_helper_spec.rb

```diff
@@ -0,0 +1,153 @@
+require_relative './spec_helper'
+
+describe Fastlane::WPMRT::AppSizeMetricsHelper do
```

Comment:
I think `WPMRT` is an alternative to `Wpmreleasetoolkit`, but it's not an alias, right? I'm guessing you chose it because it's shorter to type? _FWIW IMHO it also_ looks _better_.

Given this library is a collection of tools, of which only a portion interacts together, I think it's okay to use a different namespace. But I wonder if you have any plan to migrate the rest of the code to this syntax.

Of course, this question is unrelated to the PR 😄 Any work on those line would be out of scope here.

Reply:
Addressed in fc90b313e29a55d4fc621e705ce636f598593bbe